dags/vm_resource.py

```diff
@@ -92,7 +93,7 @@ class GpuVersion(enum.Enum):
 
   L4 = "nvidia-l4"
   A100 = "nvidia-tesla-a100"
-  H100 = "nvidia-h100-80gb"
+  H100 = "h100-80gb-8"
```

Comment:
I think PyTorch/XLA team is using H100, has the name changed? If not, we may need another entry, i.e. XPK_H100

cc @zpcore

Reply:
`h100-80gb-8` is the machine type to host those GPUs while `nvidia-h100-80gb` is the GPU. We don't need to make the change.

Reply:
I didn't find `h100-80gb-8` is mentioned anywhere. Is this a typo?

Reply:
> I didn't find `h100-80gb-8` is mentioned anywhere. Is this a typo?

This is the keyword used in xpk. I will change the name to `XPK_H100`.